### PR TITLE
set up CODEOWNERS for js-multiformats

### PR DESCRIPTION
**Pull request by galargh**
FYI, @BigLep, you can create teams and CODEOWNERS file both from GitHub Management (if we want to contain the discussion about both to a single place).

### CODE REVIEW

File: github/multiformats.yml
Comment (galargh) [inline github/multiformats.yml:1388]:
```suggestion
        content: |
          *  @multiformats/helia-maintainers @multiformats/js-libp2p-maintainers @rvagg @Gozala
```

**Comment by SgtPooki** [comment]
@BigLep action items from yesterday:

- [ ] Codeowners
    - js-libp2p-maintainers
    - helia-maintainers
    - Rod
    - Irakli
- [ ] Message about the plan and link to ownership issue

**Comment by BigLep** [comment]
Related ownership issue: https://github.com/multiformats/js-multiformats/issues/273

@achingbrain : please take a look to see if this looks good to you

File: github/multiformats.yml
Comment (galargh) [inline github/multiformats.yml:2555]: We can codify this! We can add a rule to https://github.com/multiformats/github-mgmt/blob/master/scripts/src/actions/fix-yaml-config.ts that would look up the team members at https://github.com/orgs/ipfs/teams/helia-dev and set them here too.

File: github/multiformats.yml
Comment (achingbrain) [inline github/multiformats.yml:2561]:
```suggestion
        - 2color
```

File: github/multiformats.yml
Comment (achingbrain) [inline github/multiformats.yml:2636]:
```suggestion
        - achingbrain
        - SgtPooki
        - wemeetagain
        - 2color
```

File: github/multiformats.yml
Comment (achingbrain) [reply github/multiformats.yml:2556]: Do we still want to do this?

**Review by achingbrain — APPROVED** [review]
LGTM, pending resolving the comment about automating the synching of team members from @galargh.

**Comment by galargh** [comment]
I'm going to close this PR because it got accidentally pulled into testing of other features. I'm going to recreate and merge it after the testing's complete and then I'll report back here that the PR was merged.

**Pull request closed without merging**
